lib/SensorFusion/src/App.cpp

```diff
@@ -221,22 +246,44 @@ void App::fatalErrorHandler()
 
 /**
  * Receives sensor data for sensor fusion over SerialMuxProt channel in the order defined in SerialMuxChannels.
- * @param[in]   payload         Sensor data
- * @param[in]   payloadSize     Size of 8 sensor data
+ * @param[in] payload         Sensor data
+ * @param[in] payloadSize     Size of 8 sensor data
+ * @param[in] userData      User data provided by the application.

```

Comment:
Maybe be more descriptive? You already know that is a pointer to the App instance.